packages/gateway/src/constants.ts

```diff
@@ -31,6 +34,10 @@ export const SENTRY_DSN =
   process.env.SENTRY_DSN ||
   'https://2ed906a0ba1c4de2ae3f3f898ec9df0b@sentry.io/1774551'
 
+export const PRODUCTION = process.env.NODE_ENV === 'production'
+export const QA_ENV = process.env.QA_ENV || false
+export const JWT_ISSUER = 'opencrvs:auth-service'
```

Comment:
Same with this I think, as we're not generating tokens anymore here

Reply:
Removed.